styles.css

```diff
@@ -0,0 +1,2153 @@
+    width: 20%;
+    height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  #headline {
+    background: url('images/Header.png');
+    background-size: 100% 100%;
+    background-position: 50px;
+    background-repeat: no-repeat;
+    background-color: white;
+    top: 12.5%;
+  }
+
+  .logo-placeholder {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    width: 100px;
+    height: 40px;
+    left: 143px;
+    top: -20px;
+  }
+
+  #break {
+    display: block;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 2px;
+    position: absolute;
+    width: 100%;
+    height: 268px;
+    left: 260px;
+    top: calc(40% - 268px / 2);
+  }
+
+  .primary-text1 {
+    width: 100%;
+    height: 122px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .primary-text2 {
+    width: 58%;
+    height: 98px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .socialmedia {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 16px;
+    width: 30%;
+    height: 24px;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .socialmedia-icons {
+    position: absolute;
+    width: 170.17px;
+    height: 20px;
+    left: 260px;
+    top: 430px;
+  }
+
+  .twitter {
+    position: absolute;
+    left: 1.06%;
+    right: 80.56%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .linkedin {
+    position: absolute;
+    left: 18.89%;
+    right: 70.13%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .m {
+    position: absolute;
+    left: 40.54%;
+    right: 75.07%;
+    top: 20.57%;
+    bottom: 86.12%;
+  }
+
+  .github {
+    position: absolute;
+    left: 63.32%;
+    right: 72.43%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .vector {
+    position: absolute;
+    left: 85.89%;
+    right: 70.13%;
+    top: 12.53%;
+    bottom: 87.08%;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto;
+    align-items: center;
+    justify-content: center;
+    gap: 0;
+    width: 100%;
+
+    /* height: 2788px;
+    position: absolute; */
+    top: 40rem;
+  }
+
+  .card {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
```

Comment:
- [ ] Please stick to the Figma Design and ensure that the spaces highlighted in the below screenshot are respected 👍🏼 

- Please also apply The same Note about the **duplicated style** between the mobile version and the desktop.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221350976-4f135a9c-8a4e-47e0-9b14-41981fb7d3a2.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221350847-60932594-c5e3-451c-bec5-1f09d97a606f.png)